src/components/CoursePane/RightPane.js

```diff
@@ -14,10 +16,19 @@ const styles = {
 
 class RightPane extends PureComponent {
     toggleSearch = () => {
-        dispatcher.dispatch({
-            type: 'TOGGLE_SEARCH',
-        });
-        this.forceUpdate();
+        if(RightPaneStore.getFormData().ge != 'ANY' || RightPaneStore.getFormData().deptValue != 'ALL' || 
+            RightPaneStore.getFormData().sectionCode != "" || RightPaneStore.getFormData().instructor != ""){
```

Comment:
These need to be `!==` or it throws a warning in the console: `Expected '!==' and instead saw '!='               eqeqeq`
```suggestion
        if(RightPaneStore.getFormData().ge !== 'ANY' || RightPaneStore.getFormData().deptValue !== 'ALL' || 
            RightPaneStore.getFormData().sectionCode !== "" || RightPaneStore.getFormData().instructor !== ""){
```